sc2ts/inference.py

```diff
@@ -1042,18 +1046,20 @@ def add_matching_results(
     return ts, added_groups
 
 
-def solve_num_mismatches(k):
+def solve_num_mismatches(k, num_alleles):
```

Comment:
Better to just default num_alleles=5 so we don't have to modify existing code so much?

Reply:
Okay!